Commons/src/main/java/eu/neclab/ngsildbroker/commons/constants/NGSIConstants.java

```diff
@@ -295,4 +295,5 @@ public interface NGSIConstants {
 	public static final String NGSI_LD_SCOPE = "https://uri.etsi.org/ngsi-ld/default-context/scope";
 	public static final String NGSI_LD_SCOPE_Q = "https://uri.etsi.org/ngsi-ld/default-context/scopeQ";
 	public static final Object NGSI_LD_TENANT_SHORT = "tenant";
+	public static final String CONTENT_TYPE_HEADER = "Content-Type";
```

Comment:
No need to declare again. It is already in AppConstants constant class.

Reply:
Done